From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alistair Strachan <astrachan@google.com>
Date: Mon, 29 Oct 2018 15:20:05 -0700
Subject: ANDROID: x86: Remove a useless warning message

While we wait for a clang prebuilt with asm-goto support, we have
removed the kbuild's detection of a clang without asm-goto (the
kernel builds fine without it), so we don't need to see the same
warning about not having the compiler feature fill the kernel build
log, which potentially hides other issues.

This patch should be reverted when the new clang prebuilt lands and
we have reverted the other reverts.

Bug: 120440614
Test: make ARCH=x86_64 x86_64_cuttlefish_defconfig && make ARCH=x86_64
Change-Id: I26aee5b9dd291f27b950dee4b0c64471d972ddb5
Signed-off-by: Alistair Strachan <astrachan@google.com>
---
 arch/x86/include/asm/cpufeature.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/x86/include/asm/cpufeature.h b/arch/x86/include/asm/cpufeature.h
index 59bf91c57aa8..48670eb3b4d7 100644
--- a/arch/x86/include/asm/cpufeature.h
+++ b/arch/x86/include/asm/cpufeature.h
@@ -154,9 +154,6 @@ extern void clear_cpu_cap(struct cpuinfo_x86 *c, unsigned int bit);
  * Workaround for the sake of BPF compilation which utilizes kernel
  * headers, but clang does not support ASM GOTO and fails the build.
  */
-#ifndef __BPF_TRACING__
-#warning "Compiler lacks ASM_GOTO support. Add -D __BPF_TRACING__ to your compiler arguments"
-#endif
 
 #define static_cpu_has(bit)            boot_cpu_has(bit)
 
